portx/modules/ROOT/pages/transaction-designer.adoc

```diff
@@ -11,18 +11,21 @@ _Transaction Designer_ is an PortX feature that enables you to design the _eleme
 Transaction Designer enables you to configure these elements from a central page that displays the elements along with their completion status, and what steps may remain in order to complete them. To see how to use this feature to create a transaction that receives a Purchase Order, see  xref:apm-tutorial-td[PortX Tutorial: Designing a Transaction].
```

Comment:
c/_elements_/ **_Artifacts_**/
Add "For more information about Artifacts, see <xref to Actors, Relationships and Artifacts Page>.
Augment Glossary definition of Artifact to match the definition in Actors, Relationships and Artifacts Page.